tests/Test/Pager/Subscriber/Paginate/Callback/CallbackTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use Knp\Component\Pager\Event\Subscriber\Paginate\Callback\CallbackPagination;
+use Knp\Component\Pager\Event\Subscriber\Paginate\Callback\CallbackSubscriber;
+use Test\Tool\BaseTestCase;
+use Knp\Component\Pager\Paginator;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Test\Mock\PaginationSubscriber as MockPaginationSubscriber;
+use Knp\Component\Pager\Pagination\PaginationInterface;
+
+class CallbackTest extends BaseTestCase
```

Comment:
please make class final

Reply:
none of test suite is final now, it would be inconsequential

Reply:
We need to start somewhere...

Reply:
all right, introduced